src/components/FeatureViews/Accounts/TabComponents/Schedule.js

```diff
@@ -8,10 +10,37 @@ class Schedule extends Component {
         };
     }
 
+    // get instructors schdeule and parses it into one array of objcts 
+    getInstructorSchedule = () => {
+        let schedule = this.props.work_hours
+        let key = Object.keys(this.props.work_hours)
+
+        let instructorsSchedule = key.map((iKey) => {
+            return schedule[iKey]
+        })
+        let finalInstructorSchedule = []
+        instructorsSchedule.forEach((iList) => {
+            finalInstructorSchedule = finalInstructorSchedule.concat(Object.values(iList))
+        })
+        return instructorsSchedule
+    }
```

Comment:
This entire function can be reduced to one thing: `Object.values(this.props.work_hours);` The whole thing can be written as:

    getInstructorSchedule = () => Object.values(this.props.work_hours)

(since there is no function body (`{}`), the value after the arrow is automatically returned)

Alternatively, directly pass the `Object.values` call into FullCalendar and remove the function entirely.